content/docs/command-reference/plots/index.md

```diff
@@ -1,57 +1,56 @@
 # plots
 
-A set of commands to visualize and compare _plot metrics_:
+A set of commands to visualize and compare _plot data_:
 [show](/doc/command-reference/plots/show),
-[diff](/doc/command-reference/plots/diff), and
-[modify](/doc/command-reference/plots/modify).
+[diff](/doc/command-reference/plots/diff),
+[modify](/doc/command-reference/plots/modify), and
+[templates](/doc/command-reference/plots/templates).
 
 ## Synopsis
 
 ```usage
-usage: dvc plots [-h] [-q | -v] {show,diff,modify} ...
+usage: dvc plots [-h] [-q | -v] {show,diff,modify,templates} ...
 
 positional arguments:
-  COMMAND
-    show         Generate plot from a metrics file.
-    diff         Plot differences in metrics between commits.
-    modify       Modify display properties of data-series plots (has no effect on image-type plots).
+  {show,diff,modify,templates}
+                        Use `dvc plots CMD --help` to display command-specific help.
+    show                Generate plots from target files or plots definitions from `dvc.yaml` file.
+    diff                Show multiple versions of plot data by plotting them in a single image.
+    modify              Modify display properties of data-series plot outputs (has no effect on image-type plots).
+    templates           Write built-in plots templates to a directory (.dvc/plots by default).
```

Comment:
would be better to have `dvc plots templates` (show available templates, including custom?), `dvc plots template --dump <template-name>` dumps it. 

(feels like dumping a template is an aux command compared to show / diff  and even modify)

Reply:
We can discuss and modify the usage, but I would prefer to avoid it in this PR.